pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -106,7 +107,7 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) (*apps.Deployment, error)
 	// Replace default unreachable node toleration
 	k8sutil.AddUnreachableNodeToleration(&podSpec.Spec)
 
-	if c.spec.Network.IsHost() {
+	if isHost := c.spec.Network.IsHost(); isHost {
```

Comment:
no need for the var now, right?
```suggestion
	if c.spec.Network.IsHost() {
```